REQUIRE

```diff
@@ -1,4 +1,5 @@
-julia 0.6
+julia 0.7-
```

Comment:
Try bumping this higher, like `0.7.0-beta`. That might make FemtoCleaner a little more ambitious in what it chooses to fix.

Reply:
too late I did it myself.

Reply:
yee haw

Reply:
much easier to fix deprecation warnings than do my _real_ work